### PR TITLE
ON-11090 # Update OrganisationsClient.Get(), update FormsApp types

### PR DESCRIPTION
**Pull request by kizaonline**

### CODE REVIEW

File: OneBlink.SDK/models/Organisation.cs [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
+        [JsonProperty]
+        public DateTime createdAt { get; internal set; }
+        public string trialExpiry {get;set;}
+        public List<string> tags {get;set;}
+        public bool retainSubmissionData {get;set;}
+        public long? submissionDataRetentionDays {get;set;}
+        public bool retainPrefillData {get;set;}
+        public long? prefillDataRetentionDays {get;set;}
+        public bool retainDraftData {get;set;}
+        public List<FormsAppBase> solutions {get;set;}
```
Comment (mymattcarroll) [inline OneBlink.SDK/models/Organisation.cs:22]: This is the list of types, not a list of forms apps

File: OneBlink.SDK/models/FormsAppBase.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
+using System;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OneBlink.SDK.Model
+{
+    public class FormsAppBase
+    {
+        private string[] AllowedTypes = new string[]
```
Comment (mymattcarroll) [inline OneBlink.SDK/models/FormsAppBase.cs:10]: Can this be an ENUM? then we dont need to validate the string in the setter.

Comment (kizaonline) [reply OneBlink.SDK/models/FormsAppBase.cs:10]: No, Enums have to be integral values https://stackoverflow.com/questions/8588384/how-to-define-an-enum-with-string-value

Comment (mymattcarroll) [reply OneBlink.SDK/models/FormsAppBase.cs:10]: Seems this is possible: https://stackoverflow.com/questions/2441290/javascriptserializer-json-serialization-of-enum-as-string

Comment (mymattcarroll) [reply OneBlink.SDK/models/FormsAppBase.cs:10]: worth doing IMO.

I wont disagree if you don't want to change it though :)

**Pull request merged**